### PR TITLE
chore(core): solved various minor scrutinizer issues

### PR DESCRIPTION
**Pull request by jdalsem**

### CODE REVIEW

File: actions/admin/user/delete.php [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 // forward to user administration if on a user's page as it no longer exists
 $forward = REFERER;
-if (strpos($_SERVER['HTTP_REFERER'], $username) != false) {
+if (strpos($_SERVER['HTTP_REFERER'], $username) > 0) {
```
Comment (jeabakker) [inline actions/admin/user/delete.php]: shouldn't it be `!== false`

File: engine/classes/Elgg/Application/CacheHandler.php [hunk 2]
```diff
@@ -52,6 +52,9 @@ class CacheHandler {
 
 	/** @var Request */
 	private $request;
+	
+	/** @var string */
```
Comment (jeabakker) [inline engine/classes/Elgg/Application/CacheHandler.php]: isn't this a `bool`

File: engine/classes/Elgg/Database/RelationshipsTable.php [hunk 3]
```diff
@@ -401,7 +401,7 @@ public function getEntities($options) {
 	public function getEntityRelationshipWhereSql($column, $relationship = null,
 			$relationship_guid = null, $inverse_relationship = false) {
 
-		if ($relationship == null && $relationship_guid == null) {
+		if ($relationship === null && $relationship_guid === null) {
```
Comment (jeabakker) [inline engine/classes/Elgg/Database/RelationshipsTable.php]: why not use `isset`

**Review by jeabakker — DISMISSED** [review]

File: actions/admin/user/delete.php (hunk 1 above)
Comment (hypeJunction) [reply actions/admin/user/delete.php]: I think a better way would be to explode the path and check individual segments

Comment (jeabakker) [reply actions/admin/user/delete.php]: but what if it was in a query part? checking for presence is how it works now, let't not change it too much when trying to fix a syntax error

**Pull request merged**